nailgun/entities.py

```diff
@@ -4249,17 +4319,18 @@ def path(self, which=None):
                 'smart_variables',
                 'module_streams',
                 'disassociate',
+                'traces',
+                'traces/resolve',
         ):
             return '{0}/{1}'.format(
                 super(Host, self).path(which='self'),
                 which
             )
-        elif which in ('bulk/install_content',):
-            return '{0}/{1}'.format(
-                super(Host, self).path(which='base'),
-                which
-            )
-        elif which in ('bulk/add_subscriptions',):
+        elif which in ('bulk/install_content',
+                       'bulk/add_subscriptions',
+                       'bulk/remove_subscriptions',
+                       'bulk/traces',
+                       'bulk/resolve_traces'):
```

Comment:
This removal is effect of conflict, let me know if I messed it up.

Reply:
Method `bulk_remove_subscriptions` is missing in the cherry-pick. Look at the path `'bulk/remove_subscriptions'`.

https://github.com/SatelliteQE/nailgun/blob/49f3c4fa27b1cc3f1f4081e152d37cee722e2488/nailgun/entities.py#L4206-L4221

Reply:
As the PR is related to Tracer, it would be probably better to introduce the missing method in other PR.

Reply:
Thanks for pointing that out. I see this method was added in #474, which has not been cherry-picked yet. I removed reference to that endpoint in this PR.